__tests__/__fixtures__/data/test_nested_reveal_full_document.json

```diff
@@ -0,0 +1,9 @@
+{
+  "@context": [
+    "https://www.w3.org/2018/credentials/v1",
+    "https://www.w3.org/2018/credentials/examples/v1",
+    "https://www.w3id.org/security/v3-unstable"
+  ],
+  "type": ["VerifiableCredential", "UniversityDegreeCredential"],
+  "@exlicit": true
```

Comment:
This is spelt wrong and probably not needed as the default is false when you are intending to reveal a whole frame